lib/Internal/Placeholder.php

```diff
@@ -3,7 +3,8 @@
 namespace Amp\Internal;
 
 use Amp\Failure;
-use Interop\Async\{ Loop, Promise };
+use Interop\Async\Loop;
```

Comment:
That's annoying… probably isn't worth grouping them then unless there's 3 or more.